packages/react-components/react-accordion/stories/src/Accordion/AccordionWithIcon.stories.tsx

```diff
@@ -33,3 +33,24 @@ WithIcon.parameters = {
     },
   },
 };
+
+export const WithLongHeaderText = () => (
```

Comment:
We may not need a separate story for this on the storybook website. I suggest verifying if this scenario is already covered by the Visual Regression tests [here](https://github.com/microsoft/fluentui/blob/master/apps/vr-tests-react-components/src/stories/Accordion/Accordion.stories.tsx), and if not, adding this story there.